checkout.md

```diff
@@ -0,0 +1,225 @@
+    In particular, we would like checked against the actual lesson content,
+    and would also like them labelled according to the levels of Bloom's Taxonomy.
+
+The lessons you may submit to are:
+
+1.  [Data Carpentry lessons](http://www.datacarpentry.org/lessons/):
+    we would prefer contributions to these right now.
+    *   R-ecology: <https://github.com/datacarpentry/R-ecology/>
+    *   Python-ecology: <https://github.com/datacarpentry/python-ecology/>
+    *   SQL-ecology: <https://github.com/datacarpentry/sql-ecology/>
+    *   OpenRefine-ecology: <https://github.com/datacarpentry/OpenRefine-ecology/>
+2.  [New Software Carpentry Python lesson](https://github.com/swcarpentry/python-novice-gapminder/):
+    This lesson is an experiment to see if we can start collaboration much earlier than we have in the past.
+    If you want to add to this, please submit an exercise that uses the Gapminder data set (in the "data" directory).
+    Please don't worry too much about formatting, and please only submit the Markdown files (not generated HTML) in pull requests.
+3.  [Stable Software Carpentry lessons](https://software-carpentry.org/lessons/):
+    we are currently accepting solutions to existing exercises, diagrams, and improvements to learning objectives.
+
+## Part 2: Take Part in An Online Discussion Session
+<a name="discussion-session"></a>
+
+After you have submitted a change to one of our lessons,
+the next step is to take part in an hour-long online discussion session with other trainees
+led by an experienced instructor. 
+This session gives you a change to familiarize yourself with a particular lesson
+and clarify anything that may be unclear about our operations in general.
+
+To prepare for this session,
+pick a particular lesson such as Data Carpentry's lesson on OpenRefine
+or the Software Carpentry lesson on Python
+and read the *whole* lesson.
+As you are reading,
+make notes about the following:
+
+1. Did you feel like you would like to change the flow or present the concepts differently?
+   If so, what and why?
+2. Was there anything that you found unclear in the material?
+   Did you get stuck anywhere?
+3. What are some misconceptions that the students could develop when learning this material?
+   Which bits of the module may be tricky and if poorly explained, they may lead the students to broken mental model? 
+4. What prior knowledge do learners need to have before participating in this module (whole lesson)?
+   If you were to list any prerequsites, what would they be? 
+
+The first two questions are intended to help you become more comfortable with
+[our collaborative lesson development methods]({{site.site}}/blog/2014/03/collaborative-lesson-development.html).
+The third and fourth question refer back to the instructor training course,
+and are meant to give you a chance to apply some of what you've learned to these specific lessons.
+
+Once you are familiar with a lesson,
```

Comment:
Turn this into bullet points for easier reading? I think all these are relevant so even though we'd like to reduce the amount of text, I can't see anything to take out here :-(